app/components/mandatarissen/mandataris-card.hbs

```diff
@@ -16,7 +16,9 @@
             {{this.status}}
           </AuPill>
         {{else}}
-          <p>Niet beschikbaar</p>
+          <AuPill>
+            Niet beschikbaar
+          </AuPill>
```

Comment:
Updated it to a pill 
![image](https://github.com/user-attachments/assets/d0de2b0d-5122-496c-8ddb-83c0449b1e7c)